charts/manila-csi-plugin/README.md

```diff
@@ -0,0 +1,14 @@
+# Manila CSI provisioner
```

Comment:
Just a nit here: neither Cinder or Manila CSI plugins are just provisioners, they have other competencies too. I realize that Cinder's helm chart readme was used as a template for Manila's, but it's factually not correct. This is not a blocker though, we can cook up better docs for both later.

Reply:
I am going by the docker image tag. I dont see any images tagged 0.9.0 here: https://hub.docker.com/r/k8scloudprovider/manila-csi-plugin/tags

Reply:
@brtknr right, but there's no `1.2.0` either ;) That's why I was asking what does `version: 1.2.0` relate to. If that's the chart version, then it seems a bit arbitrary. Maybe the plugin's current version (0.9.0) could pose as a starting point for chart's versioning.

Reply:
Oh chart version 1.0.0 == tag v1.16.0, Oh chart version 1.1.0 == tag v1.17.0 and 1.2.0 == tag v1.18.0.

Reply:
Normally chart version gets bumped up based on reconfiguration of the chart, e.g. new default value include new default appVersion. appVersion corresponds to the app underneath, which i presumed was the manila_csi_tag

Reply:
@brtknr Alright, fair enough.

Reply:
@brtknr IMO appVersion :1.18.0 is bit confusing , we are working on master branch, so it should be latest , on the release-branch 1.18 we can update the appversion as 1.18.0 , that maintains consistency with the manifests (and can be updated on par when we support new sidecars) . wdyt?

Reply:
Hi @ramineni, thanks for the review. It is difficult to release the `latest` tag because it will quickly go out of date. This pattern of appVersion matching the latest stable version is what a lot of projects do. If a user wants to, they can easily replace the image tag with latest via Helm CLI but it makes more sense for the default behaviour to be stable. Here is the index.yaml produced by chart-releaser action on this PR: https://brtknr.github.io/cloud-provider-openstack/index.yaml as an example of what it looks like. The `latest` tag is a moving target, it is better to be explicit about the version.

Reply:
@brtknr How do you suggest to maintain consistency between manifests and helm values . Whenever we do support new sidecar versions on master branch , we cant update the corresponding values in helm charts because that relies on 1.18 (or stable version) . And in future if we would like to use the helm in CI as well , we wont be able to do that , as that needs latest?
What we follow as of now , when we cut a stable branch (release-1.18), we update the tags to 1.18 to reflect the latest stable version and master will be always on latest.

Reply:
Hi @ramineni I have updated the chart so that new releases will use the "latest" tag. I am not sure myself whether this is good practice but hey ho. I have not checked whether stable branches can have their own workflow. We may be talking about different things here. The aim of this PR is to use Github actions to release a chart. At the moment, there is no relationship between chart version and app version. Therefore I am not convinced that updating the values in the stable branch will help us here. However this approach means that if users want to deploy a particular version of cinder/manila-csi-plugin container, they will be able to override the imageTag value to the version they choose instead of running the latest.

Reply:
@

> @brtknr Can you update? File moves etc are fine. We just miss the CI stuff.
> We can mimic [kubernetes/autoscaler#3341](https://github.com/kubernetes/autoscaler/pull/3341) .

Done @strigazi. Do you agree with the latest changes?

Reply:
@brtknr I was more of talking about compatibility of sidecars we support in actual and that of helm would be different if we make it 1.18.0 . Never mind, this PR is for github workflows , so this shouldn't be blocker.